### ISSUE TITLE
Change default Deck Load location

### ISSUE THREAD

**Issue by kellyelton** [octgn/OCTGN#1766]
Since closing #1760, it seems like we should change our default deck load location.

# Requirements
Create `DeckLoadLocation` setting.
Default `DeckLoadLocation` should be `My Documents`.
Loading a deck should set `DeckLoadLocation` to the folder the deck was loaded from.
`DeckLoadLocation` changes should persist between restarts.

**Comment by brine** [comment]
I think we still want the default deck location to be `octgn/decks`, as thats where the user saves and loads their own custom decks from.

**Comment by kellyelton** [comment]
Yeah it is currently. I'm suggesting we get away from that. Let the user put stuff wherever, we really only used that folder for installed game decks. Since we that's changing, it's original purpose is gone, and maybe we could switch to a more traditional way of saving/loading user data.

**Comment by brine** [comment]
i can get on board with that.  What about having the location be game-specific?

**Comment by kellyelton** [comment]
We already have code for game specific settings, so that could be doable aslo.

**Comment by kellyelton** [comment]
Seems to make sense.

Thanks,
*Kelly Elton*
http://www.kellyelton.com
